src/main/java/net/syscon/elite/api/resource/BookingResource.java

```diff
@@ -507,6 +511,19 @@ ReasonableAdjustments getReasonableAdjustments(@ApiParam(value = "The offender b
                                                           @ApiParam(value = "Comma separated list of one or more of the following fields - <b>eventDate, startTime, endTime, eventLocation</b>") @RequestHeader(value = "Sort-Fields", required = false) String sortFields,
                                                           @ApiParam(value = "Sort order (ASC or DESC) - defaults to ASC.", defaultValue = "ASC") @RequestHeader(value = "Sort-Order", defaultValue = "ASC", required = false) Order sortOrder);
 
+    @GetMapping("/{bookingId}/visits-with-visitors")
+    @ApiOperation(value = "All visits for offender.", notes = "All visits for offender.", nickname = "getBookingVisitsWithVisitor")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "OK", response = VisitWithVisitors.class, responseContainer = "List"),
```

Comment:
I think technically the 200 response is provided automatically so you don't need to add it explicitly.  No biggie though

Reply:
Yeap, I've removed